daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/IIBase.scala

```diff
@@ -184,10 +183,14 @@ abstract class IIBase(
     if (prop.isDefined && prop.get == "edu/illinois/ncsa/daffodil/xsd/built-in-formats.xsd") {
       SDW(
         WarnID.DeprecatedBuiltInFormats,
-        "schemaLocation property uses deprecated include/import of edu/illinois/ncsa/daffodil/xsd/built-in-formats.xsd. Use org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd instead and change the dfdl:format ref to \"GeneralFormat\".",
+        "schemaLocation property uses deprecated include/import of edu/illinois/ncsa/daffodil/xsd/built-in-formats.xsd. Use /org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd instead and change the dfdl:format ref to \"GeneralFormat\".",
```

Comment:
It has been years. I think we should get rid of this feature. We need to add that to a release note if we do, because it's a feature being removed.

Reply:
Seems resonable, this PR is probably a reasonable time to remove it. I'll add it to the Deprecation/Compatibility section.